tests/test_theme.py

```diff
@@ -2,7 +2,7 @@
 import vtk
 
 import pyvista
-from pyvista import colors
+from pyvista.plotting import colors
```

Comment:
I've since restored the `pyvista.colors` name, but it might be cleaner to keep the "proper" import in the test here, not sure.